### PR TITLE
Add more things to Defense Mode

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Mods "Add mercenary hiring to Defense Mode"

#### Purpose of change
Adds some features to Defense Mode that I couldn't carry over in the original PR.

#### Describe the solution
Allow the player to hire mercenaries to help them as followers. They start with random gear and skills and cost $500 each to hire. This should help players build a better base not only with guards and extra guns but open up the use of faction camps in Defense Mode, as NPCs could possibly collect outside resources.

#### Describe alternatives you've considered
Not doing this, or having Defense Mode be a player-only mode.

#### Testing
Went into the game and hired some people. It all worked out.

#### Additional context
This was an original feature in Defense Mode, so it's not really new, but opens up new content.

This used to fix my labeler command, but someone else got to it first.

### CODE REVIEW

**Comment by Maleclypse** [comment]
The ! at the front of your label command probably means that everything except PRs touching defense mode will now be labeled as defense mode PRs.

File: .github/labeler.yml [hunk 1]
```diff
@@ -284,4 +284,4 @@
   - "src/veh_**"
  
 "Game: Defense Mode":
-  - all: [ '!data/mods/Defense_Mode/**' ]
+  - all: [ 'data/mods/Defense_Mode/**' ]
```
Comment (casswedson) [inline .github/labeler.yml:287]:
```suggestion
  - "data/mods/Defense_Mode/*"
```

**Comment by I-am-Erk** [comment]
I can't tell you how happy I am to see some work on defense mode.

**Comment by MNG-cataclysm** [comment]
@I-am-Erk I'm glad you like it, there's more to come here in a few days.

**Pull request merged**